scripts/generate_code.py

```diff
@@ -354,6 +354,11 @@ def glob(path, pattern):
     schema="more_defaults.fbs",
     prefix=swift_prefix,
 )
+flatc(
+    SWIFT_OPTS + BASE_OPTS,
+    schema="MutatingBool.fbs",
+    prefix=swift_prefix,
+)
```

Comment:
Just FYI, it seems that the flags passed here don't match the ones in the `swifttest.sh` script